scripts/netContain/README.md

```diff
@@ -1,42 +1,62 @@
-## Introduction 
+## Introduction
 
-This Directory contains scripts required to containerize Contiv and deploy it in either docker or kubernetes modes.
-The script BuildContainer.sh should be executed from the top level directory as follows 
+This directory contains scripts required to containerize Contiv and
+deploy it in docker or kubernetes modes.
+The script build_image.sh should be executed from the top level
+directory as follows:
 
 ```
-./scripts/netContain/BuildContainer.sh
+./scripts/netContain/build_image.sh
 ```
 
-After this Script successfully executes you will get a final image which is containerized contiv. The container can then be pushed into repo for deployment on a cluster. 
+After this script successfully executes you will get a final image. The
+image can then be pushed to a registry or to the Hub for deployment on
+a cluster.
 
 ## Executing Containerized Contiv
 
-   Running as Net Master
+   Running netmaster
 ```
-	docker run -itd --pid=host --net=host --name=contivNet  --privileged   -v /etc/openvswitch:/etc/openvswitch -v /etc/kubernetes/ssl:/tmp/kubernets/ssl -v /var/run/:/var/run/  -v /var/contiv:/var/contiv rajenata/contiv:0.1  -m  -p kubernetes -c etcd://127.0.0.1:2379
+  docker run -itd --pid=host --net=host --name=contivNet \
+--privileged -v /etc/openvswitch:/etc/openvswitch \
+-v /etc/kubernetes/ssl:/tmp/kubernets/ssl -v /var/run/:/var/run/ \
+-v /var/contiv:/var/contiv rajenata/contiv:0.1  -m  -p kubernetes \
```

Comment:
`rajenata/contiv:0.1`?  Since we're removing this person as MAINTAINER, we should probably use a different image name here and the other spots in this file